docs/components/text.md

```diff
@@ -48,14 +48,14 @@ configuration.
 See more examples to see configuration of alignments, anchors, baselines,
 scaling, and auto-sizing.
 
-[exampletext]: https://aframevr.github.io/aframe/examples/test/text/index.html
-[codetext]: https://github.com/aframevr/aframe/blob/master/examples/test/text/index.html
-[exampleanchors]: https://aframevr.github.io/aframe/examples/test/text/anchors.html
-[codeanchors]: https://github.com/aframevr/aframe/blob/master/examples/test/text/anchors.html
-[examplescenarios]: https://aframevr.github.io/aframe/examples/test/text/scenarios.html
-[codescenarios]: https://github.com/aframevr/aframe/blob/master/examples/test/text/scenarios.html
-[examplesizes]: https://aframevr.github.io/aframe/examples/test/text/sizes.html
-[codesizes]: https://github.com/aframevr/aframe/blob/master/examples/test/text/sizes.html
+[exampletext]: https://aframe.io/aframe/examples/test/text/index.html
+[codetext]: https://aframe.io/aframe/examples/test/text/index.html
```

Comment:
The links to [codeXXX] should point to the code otherwise they're redundant

Reply:
What I mean is that we had two links before. One pointing to the example and the other to the code. See below

[examplesizes]: https://aframevr.github.io/aframe/examples/test/text/sizes.html
[codesizes]: https://github.com/aframevr/aframe/blob/master/examples/test/text/sizes.html

2nd link is still working. In your PR now both links point to the same URL in the examples which is redundant

Reply:
My bad, I should have been clearer in my last response. I saw what you mean and updated those links with a follow-up commit to resolve that problem. 

I made the initial commit quickly late at night when I ran into the example 404's and missed the code ones, so when I replaced them I hadn't checked. I have since fixed that in [this commit](https://github.com/aframevr/aframe/pull/5150/commits/c3ce16c1962af8991d232588f22b915bcb7ac40e) which happened after your Nov 15th comment. The code on this branch includes that fix as of the 16th (should be visible below this comment thread in the github UI)

If you look at the commit from Nov 16 (the current code in this branch) it resolves that problem so that now the links are correct for example:

```
[examplesizes]: https://aframe.io/aframe/examples/test/text/sizes.html
[codesizes]: https://github.com/aframevr/aframe/blob/master/examples/test/text/sizes.html
```

Let me know if there are any other concerns!